### PR TITLE
Interface datum pooling, power monitor qdel fix.

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

File: code/modules/power/monitor.dm [hunk 1]
```diff
@@ -50,6 +50,8 @@
 	..()
 	html_machines -= src
 
+	del(interface)	//The interface has a reference to use so we delete it.
```
Comment (PJB3005) [inline code/modules/power/monitor.dm]: Yes this is a `del()` but there's no way to not do it snoflaky without datum qdelling.

File: code/modules/html_interface/html_interface.dm [hunk 2]
```diff
@@ -116,12 +116,12 @@ mob/verb/test()
 	src.height         = height
 	src.head           = head
 
-/datum/html_interface/Del()
+/datum/html_interface/Destroy()
 	src.closeAll()
 
 	html_interfaces.Remove(src)
 
-	return ..()
```
Comment (ComicIronic) [inline code/modules/html_interface/html_interface.dm]: Keep this!

Comment (PJB3005) [inline code/modules/html_interface/html_interface.dm]: The `/datum/proc/Destroy()` is a DIRECT hard del, it would fuck up pooling.

blame @eswordthecat

Comment (ComicIronic) [inline code/modules/html_interface/html_interface.dm]: Oh right, yeah.
Classic.

**Pull request closed without merging**